services/drupal/web/themes/epa_theme/js/src/external-links.es6.js

```diff
@@ -21,9 +29,15 @@ import Drupal from 'drupal';
         ) {
           external = false;
         }
-        if (allowedDomains.includes(linkElement.host)) {
-          external = false;
-        }
+
+        allowedDomains.forEach(domain => {
+          if (
+            linkElement.host === domain ||
+            linkElement.host.endsWith(domain)
```

Comment:
@jbwagner This isn't a good way to check.  Consider a domain like `pwnedclu-in.org` -- it'll match this check.  We need to check that the linked domain either exactly matches, or that it ends with, for instance `.clu-in.org` (the leading dot ensuring that we're only matching subdomains).

Reply:
@azinck added a leading `.` character to the check to make sure it finds accurate subdomains